packages/web-react/src/components/Modal/demo/ModalUniformModalOnMobile.tsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import { Button, Modal, ModalBody, ModalDialog, ModalFooter, ModalHeader } from '../..';
+
+const ModalDefault = () => {
+  const [isFirstOpen, setFirstOpen] = useState(false);
+  const [isSecondOpen, setSecondOpen] = useState(false);
+
+  const toggleFirstModal = () => setFirstOpen(!isFirstOpen);
+  const toggleSecondModal = () => setSecondOpen(!isSecondOpen);
+
+  const handleFirstClose = () => setFirstOpen(false);
+  const handleSecondClose = () => setSecondOpen(false);
+
+  return (
+    <div className="spirit-feature-modal-enable-uniform-dialog" style={{ display: 'contents' }}>
+      {/* Set `display: contents` to enable parent stack layout. */}
```

Comment:
I think you placed the comment after the <div> in the same line and prettier placed it on the second line :) So IMHO it would be better to place it before.

Reply:
Exactly, you got it right 🙂. I wrapped it all into `React.fragment`.